docker/munimap-app/munimap.conf

```diff
@@ -61,6 +67,7 @@ MAPFISH_PRINT_CREATE_URL = '%s/print/print/munimap/report' % (MAPFISH_PRINT_BASE
 PROJECT_DIR = project('.')
 
 TRANSPORT_OPERATOR = 'moBiel GmbH'
+TRANSPORT_MAX_EXTENT = (7.9, 51.7, 9.3, 52.3)
```

Comment:
what is this needed for?

Reply:
I just copied it in from the other configurations